daphne/src/lib.rs

```diff
@@ -236,6 +242,35 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// DAP version used for a task.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub enum DapVersion {
+    #[serde(rename = "v01")]
+    Draft01,
+
+    #[serde(other)]
+    Unknown,
+}
+
+impl DapVersion {
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            DapVersion::Draft01 => "v01",
+            DapVersion::Unknown => "unknown",
```

Comment:
I don't think this variant needs to be representable as a string. (See comment below.)

Reply:
I agree, I've removed the as_str() method.